run/django/README.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with Django on Google Cloud Platform on Cloud Run (fully managed)
```

Comment:
nit: could we make this title simpler?

Getting Started with Django on Cloud Run (fully managed)